### PR TITLE
Updated to final

### PR DESCRIPTION
**Pull request by shellygill**
@doneill - Could you review this PR? I have updated this existing demo to use Quartz Final, and also updated the launcher icon so that it's more distinguishable on the screen.

### CODE REVIEW

File: 2015-EDS/JumpZoom/README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Tap the menu icon to open the navigation drawer, and choose different locations
 You can use the default gestures on the map to zoom in or out and pan the map; this will cancel any current programmatic navigation.
 
 ## Licensing
-Copyright 2015 Esri
+Copyright 2016 Esri
```
Comment (doneill) [inline 2015-EDS/JumpZoom/README.md:27]: I updated the README to make it a file changed.  Doc needs to be updated to represent 100.0.0 final release and remove references to beta 2

**Review by doneill — COMMENTED** [review]
@shellygill Looks good, just need to update the README and approve the updated libs.

**Comment by shellygill** [comment]
Thanks @doneill - changes work for me. Any specific reason you updated build tools to 25.0.1 but not 25.0.2?

**Comment by doneill** [comment]
@shellygill Nope, meant for the latest so just missed patch update 😄

**Comment by shellygill** [comment]
@doneill - OK to merge as-is, or do you want me to update to latest update build tools? (I dont believe it makes any difference to the app).

**Comment by doneill** [comment]
@shellygill The app README needs to be updated to represent the correct version of SDK, so you could update the lib with that update.

**Comment by shellygill** [comment]
Ah yes, OK, updated them now.

**Comment by doneill** [comment]
Cool, good to merge!

**Pull request merged**